R/org_classify_description_texts.R

```diff
@@ -50,12 +50,13 @@ get_aws_url <- function(ein, year = 2019, form = NULL, move_global = TRUE) {
   # Turn search parameter into character vector
   ein <- ifelse(!is.character(ein), as.character(ein), ein)
 
-  # Some organizations have two object IDs
-
-  if (exists("idx") == FALSE | year != 2019) {
+  if (exists("idx") == FALSE | # If you create idx object for the first time 
+      (exists("idx") == TRUE & year != 2019) # If you want to overwrite the existing idx object 
```

Comment:
I think this will still have the effect that if you want to find values for 10 different orgs in 2018 then for every org you run this condition will be true. So the 2018 index will be reloaded and re-assigned  each time. 

Maybe what we need is to move the assignement to the import_idx function and add a global variable for current_index_year. Then if year != current_index_year, import_idx can be called which will both assign the index and change the current_index_year value.

Would that work?

Reply:
Agreed that creating the current_index_year global variable is necessary to address this problem. I will think about where we should do this after running this morning. I may be able to do this during this weekend or can take it next Monday morning.